group_vars/orangelight/staging.yml

```diff
@@ -1,132 +1,33 @@
+passenger_app_env: "staging"
+passenger_server_name: "catalog-staging.*"
+postgres_admin_user: "postgres"
+postgres_host: "lib-postgres-staging3.princeton.edu"
 rails_app_env: "staging"
-application_db_name: "{{ ol_db_name }}"
-application_dbuser_name: "{{ ol_db_user }}"
-application_dbuser_password: "{{ ol_db_password }}"
-application_dbuser_role_attr_flags: "CREATEDB"
-ol_host_name: "catalog-staging.princeton.edu"
-application_host_protocol: "https"
-ol_staging_solr_url: http://lib-solr8-staging.princeton.edu:8983/solr/catalog-staging
-rails_app_vars:
-  - name: OL_SECRET_KEY_BASE
-    value: "{{ ol_secret_key }}"
-  - name: OL_DB
-    value: "{{ ol_db_name }}"
-  - name: OL_DB_USERNAME
-    value: "{{ ol_db_user }}"
-  - name: OL_DB_PASSWORD
-    value: "{{ ol_db_password }}"
-  - name: OL_DB_HOST
-    value: "{{ ol_db_host }}"
-  - name: OL_RABBIT_SERVER
-    value: "{{ ol_rabbit_server }}"
-  - name: OL_READ_ONLY_MODE
-    value: '{{ol_read_only_mode}}'
-  - name: SOLR_URL
-    value: "{{ ol_staging_solr_url }}"
-  - name: GRAPHQL_API_URL
-    value: "{{ ol_graph_api_url }}"
-  - name: FIGGY_URL
-    value: "{{ ol_figgy_url }}"
-  - name: OL_FEEDBACK_TO
-    value: "{{ ol_feedback_to }}"
-  - name: OL_FEEDBACK_CC
-    value: "{{ ol_feedback_cc }}"
-  - name: RESERVES_CORE
-    value: "{{ reserves_core }}"
-  - name: BD_AUTH_KEY
-    value: "{{ bd_auth_key }}"
-  - name: BD_FIND_BARCODE
-    value: "{{ bd_find_barcode }}"
-  - name: SCSB_AUTH_KEY
-    value: "{{ scsb_auth_key }}"
-  - name: HONEYBADGER_API_KEY
-    value: "{{ ol_honeybadger_key }}"
-  - name: APPLICATION_HOST
-    value: "{{ ol_host_name }}"
-  - name: APPLICATION_HOST_PROTOCOL
-    value: "{{ application_host_protocol }}"
-  - name: ALMA_REGION
-    value: "{{ vault_alma_region }}"
-  - name: ALMA_BIB_READ_WRITE
-    value: "{{ vault_alma_sandbox_bib_read_write_key }}"
-  - name: ALMA_BIBS_READ_ONLY
-    value: "{{ vault_alma_sandbox_bibs_read_only_key }}"
-  - name: ALMA_CONFIG_READ_ONLY
-    value: "{{ vault_alma_sandbox_config_read_only_key }}"
-  - name: ALMA_USERS_READ_ONLY
-    value: "{{ vault_alma_sandbox_users_read_only_key }}"
-  - name: voyager_ub_id
-    value: '{{ voyager_ub_id }}'
-  - name: SMTP_HOST
-    value: "{{ ol_smtp_host | default('localhost') }}"
-  - name: SMTP_PORT
-    value: "{{ ol_smtp_port | default('1025') }}"
-  - name: ILLIAD_API_KEY
-    value: '{{ vault_ol_illiad_api_key }}'
-  - name: ILLIAD_API_BASE_URL
-    value: '{{ vault_ol_illiad_api_base_url }}'
-  - name: CLANCY_API_KEY
-    value: '{{ vault_clancy_api_key }}'
-  - name: CLANCY_BASE_URL
-    value: '{{ vault_clancy_api_base_url }}'
-  - name: BIBDATA_BASE
-    value: 'https://bibdata-staging.princeton.edu'
-  - name: ORANGELIGHT_ADMIN_NETIDS
-    value: '"{{ ol_admin_netids | join(" ") }}"'
-sneakers_workers: EventHandler
-sneakers_worker_name: orangelight-sneakers
-ol_rails_solr_url: "{{ ol_staging_solr_url }}"
-voyager_ub_id: "{{ omit }}"
+reserves_core: "reserves"
```

Comment:
@christinach can we just remove this variable (and from the other environments too)?

Reply:
yes we can remove this